libc/test/src/__support/CMakeLists.txt

```diff
@@ -140,9 +140,10 @@ add_libc_test(
     arg_list_test.cpp
   DEPENDS
     libc.src.__support.arg_list
+    libc.src.__support.macros.properties.os
 )
 
-if(NOT LIBC_TARGET_ARCHITECTURE_IS_NVPTX)
+if(NOT LIBC_TARGET_ARCHITECTURE_IS_NVPTX AND NOT LIBC_TARGET_OS_IS_WINDOWS)
```

Comment:
Please add a `TODO` with the error you're running into with `big_int` on windows. I'm okay with just skipping it for now, but I want to make sure we fix it in future.

Reply:
Updated.